noxfile.py

```diff
@@ -48,7 +48,7 @@ def format(session):
     session.install("black", "isort")
 
     session.run("isort", "--profile=black", *SOURCE_FILES)
-    session.run("black", "--target-version=py27", *SOURCE_FILES)
+    session.run("black", "--target-version=py33", *SOURCE_FILES)
```

Comment:
Fix for running lint formatter in nox, the py27 is not supported any more. We have update it in other places in the same file but this was left.